app/src/main/java/app/envelop/ui/faq/FaqActivity.kt

```diff
@@ -0,0 +1,80 @@
+package app.envelop.ui.faq
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.content.Intent
+import android.net.Uri
+import android.os.Bundle
+import android.webkit.*
+import androidx.core.view.isVisible
+import app.envelop.R
+import app.envelop.ui.BaseActivity
+import app.envelop.ui.common.MessageManager
+import kotlinx.android.synthetic.main.activity_faq.*
+import kotlinx.android.synthetic.main.shared_appbar.*
+import javax.inject.Inject
+
+class FaqActivity : BaseActivity() {
+
+  @Inject
+  lateinit var messageManager: MessageManager
+  private var hadError = false
```

Comment:
Maybe name this variable something a bit more clear like `hadErrorLoadingWebView.